torchtune/training/checkpointing/_checkpointer.py

```diff
@@ -13,13 +13,13 @@
 
 import torch
 from safetensors.torch import save_file
-from torchtune import utils
+from torchtune import training
```

Comment:
Not a huge deal (given the scope of other stuff you're doing here) but it's a little weird to me that we use `training.MODEL_KEY` etc. when all these things are literally defined in the local directory. Feels like needless indirection to me

Reply:
eh, later problem